### PR TITLE
8149/bug/layer group feature info

### PR DESCRIPTION
**Pull request by weberjavi**
## Description
The [issue](#8149) fixed on this PR avoided the info widget to be opened when the user clicked on the map if a layer group without an assigned `id` was selected (the auto assigned `id` is `'Default'`). 
The return value of the `getSelectedLayers` selector has been filtered in order to avoid `undefined` values.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix

## Issue

**What is the current behavior?**
When a layer group (whose `id` attribute value is `'Default'`) is selected on the TOC and the user clicks in the map to get the info from the selected layers inside the group the info window is not displayed.
#8149 

**What is the new behavior?**
After filtering out `undefined` values the info widget gets displayed since the `identify` epic flow does not error. 

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/selectors/__tests__/layers-test.js [hunk 1]
```diff
@@ -801,4 +802,51 @@ describe('Test layers selectors', () => {
         const props = getAdditionalLayerFromId(state, 'layer_001');
         expect(props.id).toBe('layer_001');
     });
+
+    // eslint-disable-next-line no-only-tests/no-only-tests
```
Comment (allyoucanmap) [inline web/client/selectors/__tests__/layers-test.js:806]:
```suggestion
```

**Review by allyoucanmap — APPROVED** [review]
LGTM, I just removed the eslint comment

**Comment by allyoucanmap** [comment]
@ElenaGallo please test this on DEV and let us know if we can backport to stable branch 2022.01.xx

**Comment by ElenaGallo** [comment]
Test passed, @weberjavi please backport this pr to stable branch 2022.01.xx. Thank you

**Pull request merged**